src/components/notes/List/NoteRow.jsx

```diff
@@ -76,7 +78,11 @@ const NoteRow = ({ note, f, t, client }) => {
                 time: f(note.updated_at, 'HH:mm')
               })}
             </TableCell>
-            <TableCell className={styles.tableCell}>—</TableCell>
+            <TableCell className={`${styles.tableCell} u-flex-shrink-0`}>
+              <a href={`#/folder/${dir_id}`}>
```

Comment:
`<Link` also should we open in a new tab by default? cc @poupotte

Reply:
It's a good point. If it's possible, yes. It would be better in a new tab.

Reply:
Sure thing. Makes sense as it is not in the same SPA (slow routing)